### DISCUSSION TITLE
Invite users to create project [Oct 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4215] [Discord Threads]
> 🙋‍♂️ **prasanth**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Users can sign up directly if you share base link  
#  
> 👨‍🦲 **dstala**  
#### that wouldn't help?  
#  
> 🙋‍♂️ **prasanth**  
#### no <@857998597474484295>. i have disabled signup  
#  
> 👨‍🦲 **dstala**  
#### ok. invite users to a project is the only available option then. you can request feature in GitHub  
#  
> 💁 **o1lab**  
#### hey Prasanth  
#  
> 💁 **o1lab**  
#### you have been such a valuable contributor to our community  
#  
> 💁 **o1lab**  
#### we would love to send nocodb swag away  
#  
> 💁 **o1lab**  
#### https://forms.gle/LdFSGXknYHLGnxFF8  
#  
> 💁 **o1lab**  
#### please help fill the form  
#  
> 💁 **o1lab**  
#### and also I would love to speak with you - Im from India (karnataka)    
#### And currently in UK  
#  
> 💁 **o1lab**  
#### https://calendly.com/nocodb-meeting  
#  
> 💁 **o1lab**  
#### pls book a time from the above  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1028271731615477790)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
